libdnf5/base/goal.cpp

```diff
@@ -170,6 +172,12 @@ class Goal::Impl {
     void set_exclude_from_weak(const std::vector<std::string> & exclude_from_weak);
     void autodetect_unsatisfied_installed_weak_dependencies();
 
+    GoalProblem add_replay_to_goal(
+        base::Transaction & transaction,
+        const transaction::TransactionReplay & replay,
+        GoalJobSettings settings,
+        std::filesystem::path replay_location = "");
```

Comment:
May I ask you to somehow explain the relation between `replay_location` and `package_replay.package_path`.

Reply:
Added a comment.